### PR TITLE
Implement push state for graph

### PR DESCRIPTION
**Pull request by J0pan**
## Context
---

Please include a summary of the change and which ticket it addresses. Please also include relevant motivation and context. List any dependencies that are required for this change.

Implemented code to push states through components, graph component now has access to financial information and a list of selected cars.

This helps with ticket SE-115

### Description

- feature 1: \compare component can now access the financial information and selected cars to graph.

### Type of change

Please select options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Can This Be Tested?
---

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and any other tests you think may be worth running (help out the tester pretty please would be appreciated)

- [x] Test A - If you go through the application (fill out financial info etc) the data will be in the console as soon as you hit the compare page.

## Notes:
---

## Checklist:
---

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have given testing instructions for the reviewer 
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Jira ticket number in PR title

### CODE REVIEW

File: src/components/Compare/index.js [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+    console.log(finance)
+
+    //*****How to access the financial information:*****
+    //Address: finance.address
+    //loanAmount: finance.loanAmount,
+    //monthlyBudget: finance.monthlyBudget,
+    //downPayment: finance.downPayment 
+    //etc.
+
+    //*****How to access the list of cars selected by user:*****
```
Comment (sophiaabolore) [inline src/components/Compare/index.js:25]: Is the list obtained from this just containing the carID? What other info is included?

File: src/components/Compare/index.js [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+    const location = useLocation();
+    const { state } = location;
+    const finance = state.inputData;
+    const carList = state.selected;
+    console.log(carList)
+    console.log(finance)
+
+    //*****How to access the financial information:*****
+    //Address: finance.address
+    //loanAmount: finance.loanAmount,
```
Comment (sophiaabolore) [inline src/components/Compare/index.js:20]: Does this deal with the Ammoritization data?

File: src/components/Compare/index.js (hunk 1 above)
Comment (J0pan) [reply src/components/Compare/index.js:25]: Yeah, just the carID for the car information we have to call the senso api.

File: src/components/Compare/index.js (hunk 2 above)
Comment (J0pan) [reply src/components/Compare/index.js:20]: Not sure what you mean by the amortization data, this just has all the user's financial information.

**Pull request merged**